development/file_formats/tscn.rst

```diff
@@ -1,84 +1,87 @@
 TSCN file format
 ================
 
-A :code:`.tscn` File format is the "Text SCeNe" file format and represents
-a single scene-tree inside Godot. TSCN files have the advantage of being
-nearly human-readable and easy for version control systems to manage. During
-import the TSCN files are compiled into binary :code:`.scn` files stored
-inside the .import folder. This reduces the data size and speed up loading.
+The TSCN (text scene) file represents a single scene tree inside
+Godot. TSCN files have the advantage of being mostly human-readable and easy for
+version control systems to manage. During import, TSCN files are compiled into
+binary ``.scn`` files stored inside the .import folder. This reduces the data
+size and speeds up loading.
 
-The :code:`.escn` file format is identical to the TSCN file format, but is used to
+The ``.escn`` file format is identical to the TSCN file format, but is used to
 indicate to Godot that the file has been exported from another program and
 should not be edited by the user from within Godot.
 
-For those looking for a complete description, the parsing is handled in the
-file `resource_format_text.cpp <https://github.com/godotengine/godot/blob/master/scene/resources/resource_format_text.cpp>`_
-in the class :code:`ResourceFormatLoaderText`
+For those looking for a complete description, the parsing is handled in the file
+`resource_format_text.cpp <https://github.com/godotengine/godot/blob/master/scene/resources/resource_format_text.cpp>`_
+in the ``ResourceFormatLoaderText`` class.
 
 File structure
 --------------
 
-There are five main sections inside the TSCN File:
+There are five main sections inside the TSCN file:
 
 0. File Descriptor
 1. External resources
 2. Internal resources
 3. Nodes
 4. Connections
 
-The file descriptor looks like :code:`[gd_scene load_steps=1 format=2]` And
-should be the first entry in the file. The load_steps parameter should (in
-theory) be the number of resources within the file, though in practice its
-value seems not to matter.
+The file descriptor looks like ``[gd_scene load_steps=1 format=2]`` and should
+be the first entry in the file. The ``load_steps`` parameter should (in theory)
+be the number of resources within the file. However, in practice, its value seems
+not to matter.
```

Comment:
I know this is existing text, but if the value seems not to matter, then there should be an explanation as to why it's stored at all.

Reply:
I'm not sure what it's used for either. I guess it's used to display a progress bar with ResourceInteractiveLoader. However, if its number is incorrect, the resource will still load correctly (but the progress bar may be wrong).